database/CorePartitionManager.java

```diff
@@ -42,12 +45,12 @@
 import static com.vaticle.typedb.core.graph.common.Storage.Key.Partition.STATISTICS;
 import static com.vaticle.typedb.core.graph.common.Storage.Key.Partition.VARIABLE_START_EDGE;
 
-abstract class RocksPartitionManager {
+public abstract class CorePartitionManager {
```

Comment:
Relax the access modifier of the class, inner classes, methods, and fields. This is to allow classes from other packages to subclass and / or access them.